app/public/js/appRoutes.js

```diff
@@ -1,21 +1,59 @@
-angular.module('appRoutes', []).config(['$routeProvider', '$locationProvider',
-    function($routeProvider, $locationProvider) {
-        $routeProvider
+var app = angular.module('appRoutes', ['firebase']);
 
-        .when('/login', {
-          templateUrl: 'views/login.html',
-          controller: 'LoginController'
-        })
-    		.when('/patients', {
-    			templateUrl: 'views/patients-liste.html',
-    			controller: 'PatientsListController'
-    		})
-        .when('/patient/:id', {
-          templateUrl: 'views/patient-information.html',
-          controller: 'PatientInformationController'
-        })
-        .otherwise({redirectTo:'/login'});
+// Documentation On User Auth: https://github.com/firebase/angularfire/blob/master/docs/guide/user-auth.md
```

Comment:
Actually, this is only useful on Routes, but it could be useful somewhere else. I think it deserves its own file, even if it'll be only 5 lines.